types/office-js-preview/index.d.ts

```diff
@@ -9573,6 +9573,124 @@ declare namespace Office {
              */
             Removed = "removed"
         }
+        /**
+         * Specifies the category color.
+         * 
+         * [Api set: Mailbox Preview]
+         * 
+         * @remarks
+         * <table>
+         *   <tr><td>{@link https://docs.microsoft.com/outlook/add-ins/#extension-points | Applicable Outlook mode}</td><td>Compose or Read</td></tr>
+         * </table>
+         * 
+         * @beta
+         */
+        enum CategoryColor {
```

Comment:
These enum values are incorrect. We use the Presets defined by Graph here (Preset0,Preset1)
https://docs.microsoft.com/en-us/graph/api/resources/outlookcategory?view=graph-rest-1.0

Reply:
Updated